aea/crypto/ledger_apis.py

```diff
@@ -25,12 +25,17 @@
 
 from fetchai.ledger.api import LedgerApi as FetchLedgerApi  # type: ignore
 # from fetchai.ledger.api.tx import TxStatus
-from fetchai.ledger.crypto import Entity, Identity, Address  # type: ignore
+from fetchai.ledger.crypto import Identity, Address  # type: ignore
```

Comment:
let's add fetchai also to mypi ignore. Then the type: ignore can be removed throughout.